packages/core/src/RenderingEngine/VolumeViewport3D.ts

```diff
@@ -9,26 +10,25 @@ import { RENDERING_DEFAULTS } from '../constants';
  * which will add volumes to the specified viewports.
  */
 class VolumeViewport3D extends BaseVolumeViewport {
+  constructor(props: ViewportInput) {
+    super(props);
+
+    const { parallelProjection } = this.options;
+
+    const activeCamera = this.getVtkActiveCamera();
+
+    if (parallelProjection !== undefined) {
```

Comment:
So if `parallelProjection` is `null` we still want to call setParallelProjection? Perhaps use `if(parallelProjection != null)` instead?